site/content/docs/manifest/worker-service.en.md

```diff
@@ -134,6 +134,40 @@ Retention specifies the time a message will remain in the queue before being del
 <span class="parent-field">subscribe.queue.</span><a id="subscribe-queue-timeout" href="#subscribe-queue-timeout" class="field">`timeout`</a> <span class="type">Duration</span>  
 Timeout defines the length of time a message is unavailable after being delivered. Default 30s. Range 0s-12h.
 
+<span class="parent-field">subscribe.queue.</span><a id="subscribe-queue-fifo" href="#subscribe-queue-fifo" class="field">`fifo`</a> <span class="type">Boolean or Map</span>  
+The fifo defines the SQS FIFO Queue configurations for your default queue.   
+If you specify true, Copilot will create SQS FIFO Queue the default FIFO settings.
+
+```yaml
+subscribe:
+  topics:
+    - name: events
+      service: api
+      queue: # Define a topic-specific queue for the api-events topic.
+        timeout: 20s
+    - name: events
+      service: fe
+  queue: # By default, messages from all topics will go to a shared queue.
+    fifo: true
+```
+
+Alternativey, you can also specify the advanced SQS FIFO Queue configurations. 
+```yaml
+subscribe:
+  topics:
+    - name: events
+      service: api
+      queue: # Define a topic-specific queue for the api-events topic.
+        timeout: 20s
+    - name: events
+      service: fe
+  queue: # By default, messages from all topics will go to a shared queue.
+    fifo:
+      content_based_deduplication: true
+      deduplication_scope: messageGroup
+      fifo_throughput_limit: perMessageGroupId
+```
+
```

Comment:
Can we include the FIFO advanced fields here?  (`content_based_deduplication`, `deduplication_scope`, `throughput_limit`, `high_throughput`) 🥺